src/OpenTracing/NoopSpanContext.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace OpenTracing;
+
+use ArrayIterator;
+
+final class NoopSpanContext implements SpanContext
+{
+    public function getIterator()
+    {
+        return new ArrayIterator();
```

Comment:
Nice spot! I really like the fact that there is someone knowing the standard library better than anyone taking my back. Thanks!